src/ethereum/ethNetwork.js

```diff
@@ -248,11 +249,29 @@ export class EthereumNetwork {
       networkFee = nativeNetworkFee
     }
 
+    let feeRateUsed = {}
+
+    try {
+      feeRateUsed = {
+        // Convert gasPrice from wei to gwei
+        gasPrice: bns.div(
+          tx.gasPrice,
+          WEI_MULTIPLIER.toString(),
+          WEI_MULTIPLIER.toString().length - 1
+        ),
+        gasUsed: tx.gasUsed,
+        gasLimit: tx.gas
+      }
+    } catch (e) {
+      console.log(`Failed to construct feeRateUssed: ${e}`)
+    }
+
```

Comment:
Don't repeat yourself - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Turn this code into a function: `(gasPrice: string, gasUsed: string, gasLimit: string) => feeRateUsed`